python/pyspark/tests/test_statcounter.py

```diff
@@ -76,6 +76,31 @@ def test_merge_stats(self):
         self.assertEqual(stats.sum(), 20.0)
         self.assertAlmostEqual(stats.variance(), 1.25)
         self.assertAlmostEqual(stats.sampleVariance(), 1.4285714285714286)
+        for idx in range(2):
```

Comment:
Can we create a list with:

```
            stats1 = StatCounter([1.0, 2.0])
            stats2 = StatCounter(range(1, 301))
            stats = stats1.mergeStats(stats2) if idx == 1 else stats2.mergeStats(stats1)
```

and iterator with this `for`? Then I think we don't need to introduce this `idx`.